application/common/models/User.php

```diff
@@ -48,29 +49,59 @@ public function rules()
     }
 
     /**
+     * Limit what fields are returned from api calls
+     * @return array
+     */
+    public function fields()
+    {
+        return [
+            'first_name',
+            'last_name',
+            'idp_username' => function($model) {
+                return strtolower($model->idp_username);
```

Comment:
What's the reasoning behind lowercasing the idp_username?  Given the IdP we initially plan to use this with, I think that will be fine.  I just wasn't sure if that's generally true of IdP usernames, or if we would expect some to be case-sensitive.